packages/cycle-scripts/configs/webpack.config.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+// Silence webpack2 deprecation warnings
+// https://github.com/vuejs/vue-loader/issues/666
+process.noDeprecation = true
+
+const { createConfig, defineConstants, env, entryPoint, setOutput, sourceMaps, addPlugins } = require('@webpack-blocks/webpack2');
```

Comment:
Seems that many of those imports are not used anywhere, am I missing something?
Also destructuring assignments is not support on node4, i think we want to keep support both node 4 and 6